src/sentry/models/organizationmember.py

```diff
@@ -375,3 +375,14 @@ def delete_expired(cls, threshold):
         cls.objects.filter(token_expires_at__lt=threshold, user_id__exact=None).exclude(
             email__exact=None
         ).delete()
+
+    @classmethod
+    def get_contactable_members_for_org(cls, organization_id):
```

Comment:
This probably belongs on the manager, so you can do `OrganizationMember.objects.get_contactable_members_for_org(...)`